tracing-bom/build.gradle

```diff
@@ -4,4 +4,10 @@ plugins {
 
 micronautBom {
      extraExcludedProjects.add 'kotlin-tests'
+     suppressions {
+            bomAuthorizedGroupIds.put(
+                "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha", 
+                ["io.opentelemetry.javaagent", "io.opentelemetry", "io.opentelemetry.instrumentation"] as Set
+            )
+     }
```

Comment:
@melix when you are next back, this doesn't seem to work and the BOM validation still fails, any idea why?

Reply:
Looks like we need it